cicd-scripts/app_install.sh

```diff
@@ -1,51 +1,68 @@
 #!/bin/bash
 
+
+# A hack to get the environment running without ansible local env variables
+# This block of code will eventually be removed once ansible task is completed
+SPIDER_PYTHON_VERSION=3.12
+SPIDER_STAGING_URLS_API=https://staging.search.usa.gov/urls
```

Comment:
What about removing the environment name from this variable so we don't have to have two environment variables, SPIDER_STAGING_URLS_API and, i assume SPIDER_PROD_URLS_API when we can just have one SPIDER_URLS_API and set the value based on environment?